openmdao/recorders/tests/test_sqlite_reader.py

```diff
@@ -2758,6 +2758,33 @@ def test_reading_all_case_types(self):
         for case in expected_cases:
             self.assertTrue(case in all_driver_cases)
 
+    def test_abs_rel_error(self):
+        prob = om.Problem()
+        model = prob.model
+        model.add_subsystem('comp', ImplCompTwoStates())
+
+        # mda solver
+        nl = model.nonlinear_solver = om.NewtonSolver(solve_subsystems=False)
+        nl.options['maxiter'] = 2
+        nl.recording_options['record_abs_error'] = True
+        nl.recording_options['record_rel_error'] = True
+        nl.recording_options['record_solver_residuals'] = True
+        nl.add_recorder(self.recorder)
+
+        prob.setup()
+        prob.set_val('comp.y', 8.0)
+        prob.set_val('comp.z', 5.0)
+        prob.run_model()
+        prob.cleanup()
+
+        cr = om.CaseReader(self.filename)
+        case = cr.get_cases()[-1]
```

Comment:
Not a big deal maybe, but this is instantiating a Case object for every recorded case just to get the last one.
Would probably be better to do something like `cr.get_case(cr.list_cases()[-1])`

Reply:
There is only one case in the db, but I will change it.